docs/html/user_guide.rst

```diff
@@ -49,6 +49,15 @@ For more information and examples, see the :ref:`pip install` reference.
 .. _PyPI: https://pypi.org/
 
 
+Basic Authentication Credentials
+********************************
+
+pip supports basic authentication credentials. Basically, in the url there is
+a username and password separated by ``:``.
+
+``https://[username[:password]]@pypi.company.com/simple``
```

Comment:
I would also mention that usernames/passwords containing special characters will need to be percent-encoded, and provide a link or a table here with the specific characters impacted and how to quote them.